try {
  throw new Error('boom')
} catch (err) {}
